packages/desktopjs-openfin/src/openfin.ts

```diff
@@ -153,37 +153,19 @@ export class OpenFinContainerWindow extends ContainerWindow {
     }
 
     public get allowGrouping() {
```

Comment:
I believe for allowGrouping, getGroup, joinGroup and leaveGroup you can just delete completely as the base ContainerWindow returns false and is not supported.

Reply:
Good point, removed the methods from this file